tests/test_fixtures.py

```diff
@@ -17,73 +19,83 @@ def short_env(tmpdir):
     )
     return p
 
+
 @pytest.fixture
 def short_test(tmpdir):
     t = tmpdir.join("test.py")
     t.write(
-"""
+        """
 def test(env):
     assert True
 """
     )
     return t
 
+
 def test_config(short_test):
-    with pexpect.spawn(f'pytest --traceconfig {short_test}') as spawn:
+    with pexpect.spawn(f"pytest --traceconfig {short_test}") as spawn:
```

Comment:
Is Labgrid supposed to use " instead of ' ? I see both, I have been wondering

Reply:
Good point, I'll make sure these things remain untouched in this PR.

Personally, I'd prefer them to be consistent (whether it's `'` or `"`).

Reply:
I agree. I much prefer ' since it is easier to distinguish as small font sizes, but Labgrid seems to use both

Reply:
I just updated the `pyproject.toml` file to contain:

```toml
[tool.ruff.format]
quote-style = "single"
```

Actually, I'd suggest to re-format the whole code base. But this should go into a dedicated PR in my view.

In this PR, however, I already re-formatted the `tests/test_fixtures.py` file to this new formatting rule anyways.

Reply:
Okay, this does not work yet. All sources would have to be re-formatted this way before passing CI checks.

As a quick workaround I applied this setting to the `tests/test_fixtures.py` file, but removed it from the `pyproject.toml` file.